client/src/pages/Messages/ActiveChat/ActiveChat.tsx

```diff
@@ -0,0 +1,88 @@
+import { FC, useEffect } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+
+import Spinner from '../../../components/Spinner/Spinner';
+import Header from './Header';
+import Messages from './Messages';
+import Input from './Input';
+import { IConversations } from '../index';
+import { getConvoMessages } from '../../../helpers/APICalls/conversation';
+import { useSnackBar } from '../../../context/useSnackbarContext';
+import { useAuth } from '../../../context/useAuthContext';
+import { useMessage } from '../../../context/useMessageContext';
+
+export interface IMessages {
+  _id: string;
+  author: string;
+  text: string;
+  createdAt: string;
+  updatedAt: string;
+}

```

Comment:
Couldn't you move this to interfaces folder and use this both in the context and tsx files?

Reply:
I don't see this even being used in this file, remove?

Reply:
Sure, I do that